scientific-computing-with-python/learn-object-oriented-programming-by-building-a-coffee-making-machine/CoffeeMachine.py

```diff
@@ -0,0 +1,131 @@
+from time import sleep
+from typing import Dict, List
+
+class Maker:
+    '''
+    The Maker class represents the coffee maker itself. It keeps track of the water, milk,
+    and beans in the machine. Makes coffee and reports on the resources.
+    '''
+
+    def __init__(self, resources: Dict [str, int]):
+        self.resources = resources
+
+    def report(self):
+        for resource, amount in self.resources.items():
+            print(f'{resource}: {amount} ml')
```

Comment:
Added a simple fix for this one. I could've done something more complex and appropriate but I think this may be enough for a project of this magnitude.